.github/workflows/benchmarks_comment.yml

```diff
@@ -0,0 +1,91 @@
+name: Add Benchmark Results to PR
+on:
+  workflow_run:
```

Comment:
The PR description mentions using pull_request_target, but this is workflow_run - which is probably better, but I wanted to note the discrepancy in case there's a mistake to catch here.